src/Application/Migrations/16_function_ticket_dependency.sql

```diff
@@ -29,4 +29,33 @@ END
 $$
 LANGUAGE plpgsql;
 
+-- function returns a table with all encoding profiles, which depend
```

Comment:
Mh, I changed my mind. "Ticket" is not part of this query in any way, but the file name implies that. So it doesn't fit here either. I guess it's time for a 19_function_encodingprofile_dependency.sql or something like that.

Not part of this PR, but I'd propose to merge all ticket dependency related functions into a single file and this migration then to be renamed.